assets/react/v3/entries/course-builder/components/curriculum/TopicHeader.tsx

```diff
@@ -93,7 +93,7 @@ const TopicHeader = ({
 
   const handleSubmit = async (values: TopicForm) => {
     const response = await saveTopicMutation.mutateAsync({
-      ...(topic.isSaved && { topic_id: topic.id }),
+      ...(topic.isSaved && { topic_id: getIdWithoutPrefix('topic-', topic.id) }),
```

Comment:
Let's keep the `topic_id` in a local variable. No need to call the `getIdWithoutPrefix` function every time.